resources/migrations/001_update_migrations.yaml

```diff
@@ -5888,6 +5888,92 @@ databaseChangeLog:
                   name: field_id
             indexName: idx_query_field_field_id
 
+  - changeSet:
+      id: v50.2024-02-28T16:00:00
```

Comment:
You'll need to bump the id, since newer dates have made it into master before you. See `lint-migrations-file/change-set-ids-in-order?` failure.

Reply:
Yeah it's just branch is already used, so I'm waiting until the last moment to break it. :)